R/Features-constructors.R

```diff
@@ -3,8 +3,20 @@
 ##' @description
 ##'
 ##' Convert tabular data from a spreadsheet or a `data.frame` into a
-##' `Features` object.
-##'
+##' `Features` object (`readFeatures`) or a `SummarizedExperiment` 
+##' (`readSummarizedExperiment`).
+##'
+##' @usage
```

Comment:
Don't hard-code the usage section here - it will be generated automatically by roxygen in the final Rd file.